### ISSUE TITLE
Repo is empty

### ISSUE THREAD

**Issue by fabianWrede** [come-maiz/hello-snap-elopio#1]
Hi,

there is still a link to this repo on https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#5 even though it is empty.

best
Fabian

**Comment by techtonik** [comment]
Yup. Referenced from https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#5
